src/component/CountryDetails.jsx

```diff
@@ -0,0 +1,69 @@
+import React, {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
```

Comment:
And here, we need to import Link

`import {useParams, Link} from 'react-router-dom'`